string id
uint32 index

bool new_type
string type

bool new_custom_data
# Custom data
string data

bool new_pose
geometry_msgs/Pose pose

bool new_shape_or_convex
bool is_convex_hull

geometry_msgs/Pose center
Polygon polygon
Mesh mesh
